src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public static final String PATTERN = "dd.MM.yyyy";
```

Comment:
Let's make a pattern of a type DateTimeFormatter, but not String. I would be enough to compile it only once